lib/tasks/import/log.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+desc 'Import daily log from Access CSV at data/daily_log.csv'
+namespace :import do
+  task log: :environment do
+    CSV.foreach('data/daily_log.csv', encoding: 'ISO-8859-1', headers: true).with_index 2 do |row, line|
+      date = Date.parse row.fetch('Date:')
+      user_name = row.fetch('Dispatcher:')
+      if user_name.is_a? String
+        last_name, first_name = user_name.split(', ')
+        user = User.find_by name: [first_name, last_name].join(' ')
```

Comment:
It *may* be important, but that's just too bad.

The dispatcher table only has current dispatchers in it. STSPVs have been deleting users as they leave transit. So, the "who" in those cases is unknowable. There's nothing but an orphaned foreign key that is only used in in dispatcher database.